File: src/test/java/au/com/southsky/jfreesane/Truth2.java
package au.com.southsky.jfreesane;

import com.google.common.truth.DefaultSubject;
import com.google.common.truth.Subject;
import com.google.common.truth.Truth;

/**
 * Created by pierr on 06.07.2016.
 */
public class Truth2 {
    public static Subject<DefaultSubject, Object> assertThat(Object object) {
        return Truth.assertThat(object);
    }
}

File: src/test/java/au/com/southsky/jfreesane/Truth2.scala
package au.com.southsky.jfreesane

import com.google.common.truth.{DefaultSubject, Subject, Truth}

object Truth2 {
  def assertThat(bool: Boolean) = Truth.assertThat(java.lang.Boolean.valueOf(bool))

  def assertThat(int: Int) = Truth.assertThat(java.lang.Integer.valueOf(int))

  def assertThat(any: Any): Subject[DefaultSubject, AnyRef] = Truth.assertThat(any)
}
